watertap/unit_models/pressure_exchanger.py

```diff
@@ -140,6 +146,22 @@ def build(self):
             units=pyunits.dimensionless,
             doc='Pressure exchanger efficiency')
 
+        if self.config.has_mass_transfer:
+            self.solute_transfer_fraction = Var(
```

Comment:
In case there is more than one solute, perhaps you should add an index for `solute_list`. Or you could consider having `solution_transfer_fraction` instead of splitting it up by solvent and solute.

Reply:
Right, this will be more consistent with other modules as well. I updated the code to reflect this.